force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,8 +73,14 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if(num1!=null && num2!=null){
+            if(num1>num2){
+                return num1;
+            }else{
+                return num2;
+            }
+        }
+        return largestNum; // Replace null with the variable you used to store the result
```

Comment:
Good work. Check out the method Math.max() you can save yourself some code. 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_math.htm